Improve error message in the importer

tnajdek opened the pull request:
* New messages for connection/certificate problems
* Change error message color to better fit after the redesign

Not sure about the wording of the messages, especially don't like how it looks when printed at the default wizard width (below) but couldn't come up with anything better (and this should be something people very rarely see!)


<img width="712" alt="Screenshot 2024-03-29 at 18 00 53" src="https://github.com/zotero/zotero/assets/214628/cb62b4d9-e344-4f1c-a1e3-04377dbeda42">

dstillman commented:
There seems to be a race condition where the error message is removed:

https://github.com/zotero/zotero/assets/783082/e65ef712-95b9-4894-966e-ef2bf2c20992

Maybe only when Return is used to trigger the login? I didn't see it when clicking Next.

dstillman commented:
(Also I just realized that badssl.com is dead and serving expired certificates for many of the other error conditions.)

tnajdek commented:
@dstillman appreciate the feedback & tweaks here and in #3903.

I've fixed the problem with error message being removed too eagerly when confirming with keyboard and also adapted the code to work after c16f5d15, i.e., removed the new message and re-used one from the exception thrown by `http.js`:

<img width="712" alt="Screenshot 2024-04-02 at 18 37 53" src="https://github.com/zotero/zotero/assets/214628/fc855f72-5c54-4d37-8ffc-c137b7a2b1ff">
